src/models/request.js

```diff
@@ -0,0 +1,44 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+
+const requestSchema = new Schema(
+  {
+    name: {
+      type: String,
+      maxLength: [30, "Item name's max length is 30 characters"],
+      required: [true, 'Item name is required'],
```

Comment:
Here too, a better error message could be: `Request name should have max length of 30 characters only.`

Also replace "Item" in error messages with "Request" in this file.